pr_reminder.rb

```diff
@@ -69,16 +74,12 @@ def pending_days
   end
 end
 
-
-
-if ARGV.size < 1
-  print_usage
-  exit 1
-end
-
-organization  = ARGV[0]
-
+organization = ARGV[0]
 repos = Repository.all(organization)
+if ARGV.size > 1
+  reponames = ARGV[1, ARGV.size]
+  repos.select! { |repo| reponames.include?(repo.name) }
+end
```

Comment:
Well, I prefer more ruby code like
```
organization, *reponames = ARGV
repos = Repository.all(organization)
repos.select! { |repo| reponames.include?(repo.name) } unless reponames.empty?
```

which is three lines only and more redable for me.